frontend/packages/gitops-plugin/src/components/details/GitOpsDetailsController.tsx

```diff
@@ -1,69 +1,21 @@
 import * as React from 'react';
-import { TFunction } from 'i18next';
-import * as _ from 'lodash';
-import { useTranslation } from 'react-i18next';
-import { routeDecoratorIcon } from '@console/dev-console/src/components/import/render-utils';
 import GitOpsEmptyState from '../GitOpsEmptyState';
-import { WORKLOAD_KINDS, GitOpsResource, GitOpsEnvironment } from '../utils/gitops-types';
-import CommitDetails from './CommitDetails';
+import { GitOpsEnvironment } from '../utils/gitops-types';
 import GitOpsDetails from './GitOpsDetails';
-import PodRingWrapper from './PodRingWrapper';
-import TimestampWrapper from './TimestampWrapper';
 
 interface GitOpsDetailsControllerProps {
   envsData: GitOpsEnvironment[];
   emptyStateMsg: string;
   appName: string;
+  match: any;
```

Comment:
```
import { match as RMatch } from 'react-router-dom';
```

```suggestion
  match: RMatch<{ ... }>;
```
Fill the object inside it with anything that would be part of your route path. Not sure in your case there is much, but if your URL has `:ns` or `:name` or something in it, you'll want to type those here. Typically everything is string because it's part of a route.

Reply:
this file should have been removed since it's unnecessary now, sorry about that